h/browser/chrome/test/hypothesis-chrome-extension-test.js

```diff
@@ -198,44 +205,74 @@ describe('HypothesisChromeExtension', function () {
       it('injects the sidebar if the tab is active', function () {
         var tab = createTab();
         fakeTabState.isTabActive.withArgs(1).returns(true);
-        onUpdatedHandler(tab.id, {status: 'complete'}, tab);
-        assert.calledWith(fakeSidebarInjector.injectIntoTab, tab);
+        return onUpdatedHandler(tab.id, {status: 'complete'}, tab).then(
```

Comment:
We don't have this lib yet do we? I can add it and replace lots of places where we're doing this in tests, but suggest separate pr